docs/operating-scylla/procedures/cluster-management/add-dc-to-existing-dc.rst

```diff
@@ -1,13 +1,30 @@
 Adding a New Data Center Into an Existing ScyllaDB Cluster
 ***********************************************************
 
+.. caution::
```

Comment:
Adding a **caution** at the top of every procedure is overkill and makes all the docs look scary. It's also irrelevant for many (most?) users who set up a new cluster.
I suggest moving the caution to the end, with a short reference from the top of the procedure. Something like:

"Upgrading from 5.4? see here"